backend/dataall/core/vpc/services/vpc_service.py

```diff
@@ -96,7 +96,11 @@ def delete_network(uri):
     def get_environment_networks(environment_uri):
         with _session() as session:
             nets = []
-            all_nets = VpcRepository.get_environment_networks(session=session, environment_uri=environment_uri)
+            all_nets = VpcRepository.get_environment_networks(
+                session=session,
+                environment_uri=environment_uri,
+                groups=get_context().groups,
+            )
```

Comment:
calling out that this change only filters by user's `context.groups` meaning the env admin will no longer be able to see all VPCs in the environment (which was the previous behavior). Would have to add filter only if the group is not the env admin (or similar logic) to keep same behavior as earlier